web/src/main/java/org/springframework/security/web/csrf/CookieCsrfTokenRepository.java

```diff
@@ -192,4 +194,30 @@ public void setSecure(Boolean secure) {
 		this.secure = secure;
 	}
 
+	/**
+	 * Sets maximum age in seconds for the cookie that the expected CSRF token is saved to
+	 * and read from. By default maximum age value is -1.
+	 *
+	 * <p>
+	 * A positive value indicates that the cookie will expire after that many seconds have
+	 * passed. Note that the value is the <i>maximum</i> age when the cookie will expire,
+	 * not the cookie's current age.
+	 *
+	 * <p>
+	 * A negative value means that the cookie is not stored persistently and will be
+	 * deleted when the Web browser exits.
+	 *
+	 * <p>
+	 * A zero value causes the cookie to be deleted immediately therefore it is not a
+	 * valid value and in that case an {@link IllegalArgumentException} will be thrown.
+	 * @param cookieMaxAge an integer specifying the maximum age of the cookie in seconds;
+	 * if negative, means the cookie is not stored; if zero, the method throws an
+	 * {@link IllegalArgumentException}
+	 * @since 5.5
+	 */
+	public void setCookieMaxAge(int cookieMaxAge) {
+		Assert.isTrue(cookieMaxAge != 0, "cookieMaxAge is not zero");
```

Comment:
Can you please change this to be cookieMaxAge cannot be zero? As it is, the error the user will see if it is `0` would state it is not zero which would be confusing.

```suggestion
		Assert.isTrue(cookieMaxAge != 0, "cookieMaxAge cannot be zero");
```

Reply:
Thank you for your feedback. In this class there are other assertions written in this way. I've tried to follow the convention in the class. Can I change the other assertions in the same file in order to be consistent?